dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -93,4 +96,14 @@ def report_abuse_form
       age: (current_user.age unless current_user.nil?),
     }
   end
+
+  private
+
+  def check_if_featured(project_id)
```

Comment:
Even for a helper like this, I think we prefer naming with question marks for methods that return a boolean-ish value.

I guess my full recommendation is something like:

```rb
private def featured?(encrypted_channel_id)
  _, channel_id = storage_decrypt_channel_id encrypted_channel_id
  FeaturedProject.find_by(storage_app_id: channel_id)&.featured?
rescue
  false
end
```

I'd also consider whether this helper ought to live as a static method on `FeaturedProject`.

Reply:
\It might make more sense (conceptually) for the `FeaturedProject` model not to know how about encrypted channel ids, in which case I'd leave the helper here in this controller. Otherwise, if it's reasonable for the model to be aware of those ids, I agree it would be best as a class method on `FeaturedProject`